libc/include/malloc.h

```diff
@@ -29,6 +29,9 @@ extern void *(*__alloca_alloc)(size_t);
 #define malloc(x) ((*__alloca_alloc)(x))
 #endif
 
+/* request paras from main memory, returns segment */
+int _fmemalloc(int paras, unsigned short *pseg);
```

Comment:
Lets use `seg_t *` rather than `unsigned short *` to better indicate that the system call returns a segment number.

Reply:
Yes, the `seg_t` type should be picked up from types.h. This renaming is all part of an ongoing cleanup for ELKS.